dali/base/dadfs.cpp

```diff
@@ -9567,7 +9571,7 @@ class CDaliDFSServer: public Thread, public CTransactionLogTracker, implements I
         StringArray authScopes;
         CIArrayOf<CFileMatch> matchingFiles;
         start = msTick();
-        count = scanner.getResults(auth, udesc, matchingFiles, authScopes);
+        count = scanner.getResults(auth, udesc, matchingFiles, authScopes, false);
```

Comment:
It just exceeded the 100 MB buffer that was allocated for the sort.  You would get an exception with jbuf OOM ...
Kevin calculated at least 200 MB would be needed for approx 2.4 M files and it proved it worked with that.
We were thinking of a merge sort with iteration to try to eliminate a mem limit, but there is still the concept that it takes many minutes to return to the user and the number of pages to scan is too large.